tests/yam/agama/patch_agama.pm

```diff
@@ -0,0 +1,35 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: This module use yupdate patch the Agama on Live Medium
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    assert_screen('agama_product_selection', 120);
+
+    select_serial_terminal;
+
+    my ($repo, $branch) = split /#/, get_var('YUPDATE_GIT');
+    if ($repo && $branch) {
+        assert_script_run("yupdate patch $repo $branch", timeout => 60);
+        assert_script_run("playwright test --trace on --project chromium --config /usr/share/e2e-agama-playwright/ test.spec", timeout => 120);
```

Comment:
patching should not include test execution, this should go in a different test module,
something like `agama_installation` but more like `agama_dummy_test`, then in other PR, when we have `agama_installation_lvm` we can delete this test. But yeah, definitely needs to be separated from the patching.

Reply:
ignore previous comment here, thinking twice we should not repeat this test module, probably just pass a variable to use it with the test we want, Lemon wanted to do initially and now seems that makes sense.
could you create a setting that we can pass [here](https://github.com/lemon-suse/os-autoinst-distri-opensuse/blob/df94211770533cbc89d2c8317c02abe70e9d2acb/tests/yam/agama/agama_installation.pm)